src/components/ProjectIconSelection/ProjectIconSelection.js

```diff
@@ -44,11 +44,9 @@ class ProjectIconSelection extends Component<Props> {
               size={DEFAULT_ICON_SIZE}
               onClick={ev => this.props.onSelectIcon(src, ev)}
               status={
-                selectedIcon === null
-                  ? 'default'
-                  : selectedIcon.includes(src)
-                    ? 'highlighted'
-                    : 'faded'
+                selectedIcon && selectedIcon.includes(src)
```

Comment:
@superhawk610 is this OK? I think we don't need the default status string or am I missing something? I've changed it to fix flow typing & to make this easier to read.

Reply:
I think the old code is needed, because selectedIcon will be null before you have made any icon selection. A `status` of 'default' has the icon at full brightness by default. If it's not set to default, it's faded.

Old code:

<img width="200" alt="Screen Shot 2019-05-05 at 11 55 52 AM" src="https://user-images.githubusercontent.com/17421347/57198880-0f27f880-6f2d-11e9-9a68-91d4b2b8f1e3.png">

New code:

<img width="200" alt="Screen Shot 2019-05-05 at 11 57 20 AM" src="https://user-images.githubusercontent.com/17421347/57198881-13ecac80-6f2d-11e9-8c28-3c7c8e573166.png">

Reply:
Thanks @melanieseltzer. I'll revert the change so it's not faded if no icon is selected.